lib/sidekiq/throttled/web/summary_fix.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Sidekiq
+  module Throttled
+    module Web
+      module SummaryFix
+        JAVASCRIPT = <<-JAVASCRIPT
+        <script>
+          $(function ($el) {
+            var $el = $(".summary li.enqueued > a"),
```

Comment:
Yeah. This is the only path that needs fix (the other one we override much more elegantly).
If in future it will get changed - tests will fail. Travis will be running specs on daily basis, so we will know that fast.